### ISSUE TITLE
Psql can`t connect migrate with my db

### ISSUE THREAD

**Issue by EightLegs** [golang-migrate/migrate#823]
**Describe the Bug**
I read the tutorial about how to connect and use migrate with my postgresql db. But when i use "migrate create" command migration files are empty

**Steps to Reproduce**
1. I execute export with my postgres user, his password, and db service_users 
![image](https://user-images.githubusercontent.com/37411749/195028578-23ea62cb-89e4-411f-9182-908721f417f3.png)
2. the output files are empty
3. there`s no any record about my migrate creation attempt in postgresql logs

**Expected Behavior**
There should be any evidence, that i use migrate program with my db. Any record in postgresql log file, any data in output files. 
it looks like i`ve not even connected with my db.

**Migrate Version**
e.g. v4.15.2
Obtained by running: `migrate -version`

**Loaded Source Drivers**
Source drivers: s3, bitbucket, github-ee, gitlab, go-bindata, godoc-vfs, github, gcs, file

**Loaded Database Drivers**
Database drivers: stub, cassandra, firebird, neo4j, postgres, cockroach, mysql, clickhouse, cockroachdb, crdb-postgres, postgresql, redshift, spanner, sqlserver, firebirdsql, mongodb, mongodb+srv, pgx

**Go Version**
go version go1.19.2 linux/amd6

**Stacktrace**
Please provide if available

**Additional context**
Please, help me to connect to my db with this program

**Comment by bengesoff** [comment]
You'll want to look into the `migrate up` command to actually apply the migrations - the `create` command just creates the files for you to populate

**Comment by EightLegs** [comment]
> 

No no no!) I would like to create sql files with content of creation scripts for my database. In order to do it one should use `migrate create`


`migrate up` is used for  applying migrations to my db, i don`t need it right now

**Comment by bengesoff** [comment]
@EightLegs makes sense - in that case I think you need a different tool as this one is mainly good for creating your schema from scratch, rather than exporting an existing schema into migration scripts :)

**Comment by drev74** [comment]
@Vsevosemnog That's an expected behavior. Migration create adds an empty numbered `up` and `down` files, which you need to fill with actual SQL commands
